mozilla/blurts-server
# Port limits-alert cronjob to TypeScript

# References:

Jira: MNTOR-2142
Figma: N/A



# How to test

Run `npm run dev:cron:onerep-limits-alert`; you should see stats get added to your `stats` table locally and, if you exceed the thresholds, something like this:

```
Alert: OneRep scans over limit for free_scans. Current: 3392, max: 100
Alert: OneRep scans over limit for paid_scans. Current: 1372, max: 100
Alert: OneRep scans over limit for profiles_activated. Current: 1372, max: 100
Alert: OneRep scans over limit for profiles_created. Current: 3278, max: 100
```

# Checklist (Definition of Done)

- [ ] I've added a unit test to test for potential regressions of this bug. - N/A, no funcitonal changes
